detect_secrets/plugins/keyword.py

```diff
@@ -170,17 +189,23 @@ def secret_generator(self, string, filetype):
 def probably_false_positive(lowered_secret, filetype):
     if (
         'fake' in lowered_secret
-        or 'self.' in lowered_secret
-        or lowered_secret in FALSE_POSITIVES or
-        # If it is a .php file, do not report $variables
-        (
+        or 'forgot' in lowered_secret
+        or lowered_secret in FALSE_POSITIVES
+        or (
+            filetype == FileType.JAVASCRIPT
+            and (
+                lowered_secret.startswith('this.')
+                or lowered_secret.startswith('fs.read')
+                or lowered_secret == 'new'
+            )
+        ) or (  # If it is a .php file, do not report $variables
             filetype == FileType.PHP
             and lowered_secret[0] == '$'
         )
     ):
         return True
 
-    # No function calls
+    # Hueristic for no function calls
```

Comment:
nit: it's `heuristic`

Reply:
Great catch! I can `git commit --amend` it like it never happened.